src/util/hass-media-player-model.js

```diff
@@ -178,13 +182,21 @@ export default class MediaPlayerEntity {
   }
 
   togglePower() {
```

Comment:
I'm not sure about adding the toggle method(). Included as reference.

Reply:
I think, this should better do `this.callService("toggle");` and have the decision for the right action in the backend.
It's missing in different places. Otherwise external plugins will keep being affected, like kalkih/mini-media-player#218